docs/audit-log-filter/audit-log-filter-naming.md

```diff
@@ -0,0 +1,114 @@
+# Audit Log Filter file naming conventions
+
+The default name for the audit log filter file in the server directory is `audit_filter.log`. For security, the audit log filter file should be in a directory with the following access qualifications:
```

Comment:
This duplicates what is mentioned in "Audit Log Filter security"

Reply:
Removed.